unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -0,0 +1,96 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+
+import time
+import pyrealsense2 as rs
+
+from rspy import test
+
+
+def close_resources(sensor):
+    """
+    Stop and Close sensor.
+    :sensor: sensor of device
+    """
+    if len(sensor.get_active_streams()) > 0:
+        sensor.stop()
+        sensor.close()
+
+
+def set_and_verify_timestamp_domain(sensor, global_time_enabled: bool):
+    """
+    Perform sensor (depth or color) test according given global time
+    :sensor: depth or color sensor in device
+    :global_time_enabled bool: True - timestamp is enabled otherwise false
+    """
+    global frame_queue
+
+    try:
+        sensor.set_option(rs.option.global_time_enabled, 1 if global_time_enabled else 0)
```

Comment:
Can you check if we can use 
```
sensor.set_option(rs.option.global_time_enabled, global_time_enabled)
```
?

Reply:
Yes, it works! The Code changed accordingly.
The Rule: [python-booleans-as-numbers](https://realpython.com/python-boolean/#python-booleans-as-numbers)

Reply:
Nice to know :)